environment.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   - graphviz>=2.38
   - openjdk=8
   - pytables>=3.6
-  - python-graphviz>=0.14
+  - python-graphviz>=0.14, <=0.15
```

Comment:
Thanks @timothydijamco 

This could be simply `=0.14` if I'm not wrong.

But this is inconsistent with what you say in #2585, where you say we should be using `0.15` since it's `0.16` the version that fails in Windows.  Not sure if I'm missing something.

Reply:
@datapythonista the current pinning I think allows either `0.14` or `0.15` (with the latest version of this PR it's currently resolving to `0.15`). I think `0.14` _might_ be OK too (i don't have any data points on whether it works or not though)